news/461bbf24-25f6-488a-b7c0-7ff34bf4f0c1.trivial

```diff
@@ -0,0 +1 @@
+Add logger alert for when 'wheel' is not installed.
```

Comment:
To appear in the changelog, this should be named `7768.feature`.

Reply:
Updated in https://github.com/pypa/pip/pull/7768/commits/4b6842238db6e66c2f5169adf780255502e22558